include/fluidsynth/misc.h

```diff
@@ -67,6 +67,7 @@ extern "C" {
 
 FLUIDSYNTH_API int fluid_is_soundfont(const char *filename);
 FLUIDSYNTH_API int fluid_is_midifile(const char *filename);
+FLUIDSYNTH_API void* fluid_alloc(size_t len);
```

Comment:
I'm hesitant to make fluid_alloc public for everyone. It is only used inside libfluidsynth. I'd prefer that you use plain `malloc()` and `free()` in `fluidsynth.c` and do not make it public.

Reply:
I agree, and thank you for checking.
I'll replace the code with malloc and free, test it, and then make a pull request again.
Please wait a while.

Reply:
Completed.
https://github.com/stardusteyes/fluidsynth/pull/1/files

Please confirm.